src/Lykke.ClientGenerator/Caching/ClientCachingAttribute.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+
+namespace Lykke.ClientGenerator.Caching
+{
+    /// <summary>
+    /// Specifies the amount of time to cache this method call.
+    /// Used with <see cref="AttributeBasedCachingCallsWrapper"/>.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method)]
+    public class ClientCachingAttribute : Attribute
```

Comment:
Will be cosily if opportunity to initialize attribute by the string in TimeSpan fromat will be added. For example ```[ClientCaching("00:05:00")]```